docs/graphcast/radios/subgraph-radio.md

```diff
@@ -2,45 +2,39 @@
 sidebar_position: 1
 ---
 
-# 📟 POI Radio
+# 📟 Subgraph Radio
 
-The source code for the POI Radio is available [on GitHub](https://github.com/graphops/poi-radio) and Docker builds are automatically published as [GitHub Packages](https://github.com/graphops/poi-radio/pkgs/container/poi-radio). POI Radio is also published as a Crate [on Crates.io](https://crates.io/crates/poi-radio).
+The source code for the Subgraph Radio is available [on GitHub](https://github.com/graphops/subgraph-radio) and Docker builds are automatically published as [GitHub Packages](https://github.com/graphops/subgraph-radio/pkgs/container/subgraph-radio). Subgraph Radio is also published as a crate [on crates.io](https://crates.io/crates/subgraph-radio).
 
 ## Introduction
 
-The POI Radio is an optional component of the Graph Protocol Indexer Stack that leverages the Graphcast Network to cross-check subgraph data integrity with other participating Indexers.
+The Subgraph Radio is an optional component of the Graph Protocol Indexer Stack that leverages the Graphcast Network to cross-check subgraph data integrity with other participating Indexers.
```

Comment:
Please read through the Introduction section and update the narrative to more Subgraph centric.